### PR TITLE
modifiy index.md of apparmor 

### PR DESCRIPTION
**Pull request by CandiceGuo**
I have found several English grammar mistakes in file "/docs/admin/apparmor/index.md". And I have try to correct them.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1991)

### CODE REVIEW

**Comment by CandiceGuo** [comment]
I signed it

File: docs/admin/apparmor/index.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ added in Kubernetes v1.4.
 
 ## What is AppArmor
 
-AppArmor is a Linux kernel security module that supplements the standard Linux user and group based
-permissions to confine programs to a limited set of resources. AppArmor can be configured for any
+AppArmor is a Linux kernel security module that supplements the standard Linux user and group based 
+on permissions to confine programs to a limited set of resources. AppArmor can be configured for any
```
Comment (krzyzacy) [inline docs/admin/apparmor/index.md]: the grammar reads : `AppArmor` is a `Linux kernel security module` that `supplements` `the standard Linux user and group based permissions`, but not based on

**Comment by CandiceGuo** [comment]
I signed it

**Comment by krzyzacy** [comment]
https://github.com/kubernetes/kubernetes.github.io/pull/1991/commits/82b904eb95ddf880e098bbeee47478c118a49f76 and https://github.com/kubernetes/kubernetes.github.io/pull/1991/commits/1bff2c870a09dcf2662ea4293df7d67143511d9a is committed as `Candicebigcat` which is a different user, you can squash and commit everything  again.

**Comment by CandiceGuo** [comment]
How to squash and commit ?

File: docs/admin/apparmor/index.md [hunk 2]
```diff
@@ -316,7 +316,7 @@ No action is required with respect to AppArmor to upgrade your cluster to v1.4.
 existing pods had an AppArmor annotation, they will not go through validation (or PodSecurityPolicy
 admission). If permissive profiles are loaded on the nodes, a malicious user could pre-apply a
 permissive profile to escalate the pod privileges above the docker-default. If this is a concern, it
-is recommended to scrub the cluster of any pods containing an annotation with
+is recommended to scrubbing the cluster of any pods containing an annotation with
```
Comment (krzyzacy) [inline docs/admin/apparmor/index.md]: recommend to do sth.

File: docs/admin/apparmor/index.md [hunk 3]
```diff
@@ -112,7 +112,7 @@ specifies the profile to apply. The `profile_ref` can be one of:
 See the [API Reference](#api-reference) for the full details on the annotation and profile name formats.
 
 The Kubernetes AppArmor enforcement works by first checking that all the prerequisites have been
-met, and then forwarding the profile selection to the container runtime for enforcement. If the
+met, and then forwarding the profile selection of the container runtime for enforcement. If the
 prerequisites have not been met, the Pod will be rejected, and will not run.
```
Comment (krzyzacy) [inline docs/admin/apparmor/index.md]: forward foo to bar right?

File: docs/admin/apparmor/index.md [hunk 4]
```diff
@@ -13,15 +13,15 @@ added in Kubernetes v1.4.
...
+AppArmor is a Linux kernel security module that supplements the standard Linux user and group based 
+on permissions to confine programs to a limited set of resources. AppArmor can be configured for any
 application to reduce its potential attack surface and provide greater defense in depth. It is
 configured through profiles tuned to whitelist the access needed by a specific program or container,
 such as Linux capabilities, network access, file permissions, etc. Each profile can be run in either
 enforcing mode, which blocks access to disallowed resources, or complain mode, which only reports
 violations.
 
-AppArmor can help you to run a more secure deployment by restricting what containers are allowed to
+AppArmor can help you to run a more secure deployment by restricting what containers is allowed to
```
Comment (krzyzacy) [inline docs/admin/apparmor/index.md]: what containers are allowed is correct.

File: docs/admin/high-availability/index.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ Users who merely want to experiment with Kubernetes are encouraged to use config
 the simple [Docker based single node cluster instructions](/docs/getting-started-guides/docker),
 or try [Google Container Engine](https://cloud.google.com/container-engine/) for hosted Kubernetes.
 
-Also, at this time high availability support for Kubernetes is not continuously tested in our end-to-end (e2e) testing.  We will
+Also, at this time high availability supports for Kubernetes is not continuously tested in our end-to-end (e2e) testing.  We will
```
Comment (krzyzacy) [inline docs/admin/high-availability/index.md]: assume support is a none-countable noun.

**Comment by krzyzacy** [comment]
Left some comment while watching the signing, I'm not a good grammar fixer anyhow, so @mml plz take another look.

**Comment by krzyzacy** [comment]
@CandiceGuo `How to squash and commit ?` http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git answers it.

**Comment by devin-donnelly** [comment]
@CandiceGuo , this appears to be a duplicate of #1990; please close either that one or this one. Also, there appears to be an issue with your CLA that you need to resolve.

**Comment by devin-donnelly** [comment]
@CandiceGuo , I don't think any of these edits are actually necessary; I'm going to go ahead and close this PR as well as the duplicate.

**Pull request closed without merging**